tests/unit/test-cmd/test.lint.js

```diff
@@ -96,13 +104,16 @@ describe('lint', () => {
       boring: 'boring flag',
       // $FLOW_IGNORE: wrong type used for testing purpose
       selfHosted: 'self-hosted flag',
+      // $FLOW_IGNORE: wrong type used for testing purpose
+      warningsAsErrors: 'warnings-as-errors flag',
```

Comment:
I think you can remove this entirely. Or is there any reason to include it? Do the tests fail?